src/components/Notifier/index.js

```diff
@@ -156,6 +168,13 @@ const Notifier = function(props) {
                 <input type="radio" name="age" value="45" onChange={onAgeChange} checked={ageGroup === `45` ? true : false} /><label>45+</label>
             </div>
         </div>
+        <div className="row">
+            <div className="label">Dose:</div>
+            <div>
+                <input type="radio" name="dose1" value="dose1" onChange={onDoseChange} checked={dose === `dose1` ? true : false} /><label>first dose</label>
```

Comment:
the name for both radios should be same, same names identify these radios as one radio group